print.go

```diff
@@ -64,6 +65,11 @@ func Print(idProvider identityProviders.IdentityProvider, awsProvider servicePro
 	}
 
 	creds := awsProvider.GetCredentials(saml, printOptions.Role)
+
+	if printOptions.AssumeRole != "" {
+		creds, err = awsProvider.AssumeRole(*creds, printOptions.AssumeRole)
```

Comment:
I think this'll need a check on the `err`

Reply:
Probably. Do we just panic or is there a more graceful way out of here that we want to use?

Reply:
For now I think a `log.Fatal(err)` is fine, as it happens above with authenticate.